.github/workflows/main.yml

```diff
@@ -8,25 +8,18 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # >= we can make C++-17 work on bionic with the ubuntu-toolchain-r PPA
-          # use a tool like, e.g., linuxdeploy-plugin-checkrt to bundle libstdc++ when building libappimage-based packages on bionic
-          - DIST: bionic
+          - RELEASE: latest
             ARCH: x86_64
-          - DIST: bionic
-            ARCH: i386
-
-          # CentOS 7 based Docker images can use devtoolset-9 to get a compatible compiler (as demonstrated by appimagebuild)
-          # (devtoolset compilers can build ABI compatible builds by embedding parts of libc/libstdc++, so ld-p-checkrt is not needed)
-          - DIST: appimagebuild
-            ARCH: x86_64
-          - DIST: appimagebuild
-            ARCH: i386
+          - RELEASE: latest
+            ARCH: armhf
+          - RELEASE: latest
+            ARCH: aarch64
 
           # special builds
-          - DIST: bionic
+          - RELEASE: latest
```

Comment:
Pinning is recommended, because "latest" is a moving target and may break anytime.